app/javascript/controllers/token_controller.js

```diff
@@ -35,4 +35,11 @@ export default class extends Controller {
     }
     this.visible = !this.visible;
   }
+
+  removeTokenPanel() {
+    let panel = document.getElementById("access-token-section");
```

Comment:
This DOM element is outside of the current controller:
![image](https://github.com/user-attachments/assets/752110de-c785-4fbe-bab4-c5060ab53146)

I could move the controller call further 'out' but figured this would be a simpler solution without messing too much with the token controller. Thoughts?